crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -51,14 +53,23 @@ impl Violation for Airflow3Removal {
         } = self;
         match replacement {
             Replacement::None => format!("`{deprecated}` is removed in Airflow 3.0"),
-            Replacement::Name(name) => {
-                format!("`{deprecated}` is removed in Airflow 3.0; use `{name}` instead")
+            Replacement::Name(_) => {
+                format!("`{deprecated}` is removed in Airflow 3.0.")
```

Comment:
The convention is to not use a period in user facing messages, you'll need to update the snapshots

```suggestion
            Replacement::Name(_) => {
                format!("`{deprecated}` is removed in Airflow 3.0")
```

Reply:
Thanks, updated and will follow the convention in future PRs.